psclient/src/main/java/com/majeur/psclient/service/observer/BattleRoomMessageObserver.kt

```diff
@@ -867,5 +905,26 @@ class BattleRoomMessageObserver(service: ShowdownService)
         fun onSideChanged(player: Player, side: String, start: Boolean)
         fun onVolatileStatusChanged(id: PokemonId, vStatus: String, start: Boolean)
         fun onPrintBattleMessage(message: CharSequence)
+        fun onSetBattleType(type: BattleType)
+        fun goToLatest()
```

Comment:
What's the point of goToLatest() ?

Reply:
For requesting to move to the bottom of the battle log. 

If your replay is paused, then I thought it would make sense to auto-scroll to the latest message whenever you click the 'play' or 'next turn' buttons. This is only noticeable if the replay is paused and you scroll the battle log. Just a kind of quality of life thing, as it seems like if the user has chosen to manually invoke the play or next turn button then we should take them to the corresponding info in the battle log.

It's somewhat situational though, so I'm happy to remove if you think it's not necessary :+1: 

I'm not sure if I explained it very well, so here's a video just in case https://drive.google.com/file/d/1y5NN4A8RHxGO9mrltZXB2Twhv-dm25EC/view?usp=sharing